ci/k8s/client/test_fleet_scale.py

```
@@ -0,0 +1,113 @@
+
+    The source worker downloads from HF and never logs the marker; every
+    other worker must pull via NIXL (TCP transport). Counting occurrences
+    of "RDMA transfer complete" across all pod logs — each receiving worker
+    logs it exactly once — gives the number of successful P2P transfers.
+    A count below fleet_size - 1 means at least one non-source worker fell
+    back to the HF disk path, which is the failure mode this test guards.
+    """
+    all_logs = ""
+    for pod in _worker_pod_names(namespace):
+        try:
+            r = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)